staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/clientset/versioned/typed/cr/v1/cr_client.go

```diff
@@ -43,7 +43,7 @@ type CrV1Client struct {
 }
 
 func (c *CrV1Client) Examples(namespace string) ExampleInterface {
-	return newExamplesWithMultiTenancy(c, namespace, "default")
+	return newExamplesWithMultiTenancy(c, namespace, "system")
```

Comment:
Can you use constant instead of string?

Reply:
This is a file generated by codegen. It is hard to use constant like metav1.TenantSystem in the generated files, as codegen does not support adding dependency package for a constant.  codegen will add the dependent package for  a func/variable/type, but not for a const.